app/server/middleware/sso.ts

```diff
@@ -39,12 +41,11 @@ export default async function middleware() {
       sessionIdleRemainingTime: mockSessionTimeout,
     },
     oidcConfig: {
-      baseUrl:
-        process.env.HOST || `http://localhost:${process.env.PORT || 3004}`,
+      baseUrl: config.get("host"),
       clientId: "cas-cif",
       oidcIssuer: `https://${ssoServerHost}/auth/realms/pisrwwhx`,
     },
-    ...(process.env.SHOW_KC_LOGIN !== "true" && {
+    ...(config.get("showKCLogin") !== true && {
```

Comment:
```suggestion
    ...(config.get("showKCLogin") || {
```

Reply:
with the spread it has to be &&, went with `...(!config.get("showKCLogin") || {`